### ISSUE TITLE
jbuilder extra fields

### ISSUE THREAD

**Issue by elfassy** [amatsuda/active_decorator#36]
when using with a `jbuilder` partial i get:

```
:companies=>
  [{:is_a?=>{},
    :respond_to?=>"to_a_with_decorator",
    :id=>1,
    :name=>"Carter and Sons",
    :logo_url=>"/media/W1siZiIsInNvbWUvdWlkL3BhdGgiXV0?sha=9fc68caa72d460cb"}],
 :success=>true}
```

not sure why  `:is_a?=>{}` and `:respond_to?=>"to_a_with_decorator"` get added. Thoughts?

**Comment by amatsuda** [comment]
@elfassy I just released a new gem versioned 0.4.0 which includes a fix for this issue.
Could you `bundle up active_decorator` and try again?
Thanks!

**Comment by elfassy** [comment]
works, thanks!
